src/client.rs

```diff
@@ -22,11 +22,98 @@ struct Host {
     token: String,
 }
 
-/// Reads the contents of a hosts.toml file located in `dir`.
-fn read_hosts_toml(mut dir: PathBuf) -> Result<Hosts> {
-    dir.push("hosts.toml");
-    let hosts = std::fs::read_to_string(dir)?;
-    Ok(toml::from_str(&hosts)?)
+/// The contents of an Oxide CLI `credentials.toml` file.
```

Comment:
Not blocking: a small example of what the serialized file looks like would be nice, or example field values.

Reply:
Makes sense, added.